articles/techniques/index.md

```diff
@@ -12,26 +12,38 @@ uid: microsoft.quantum.techniques.intro
 
 # Quantum Development Techniques
 
-![Quantum](~/media/mobius_strip_preview.png)
-
-This section details the core concepts used to create quantum programs in Q#, and to interact with those programs from classical .NET applications.
-This section assumes some knowledge of quantum computing concepts like those described in [Quantum computing concepts](xref:microsoft.quantum.concepts.intro).
-
-
-
-
-
-
-
+This section of our documentation details the core concepts used to create quantum programs in Q#, and to interact with those programs from classical .NET applications.
```

Comment:
Do we want to keep the previous phrasing about "classical .NET applications," or do we want to broaden a bit to reflect Python host functionality and Jupyter Notebooks?
```suggestion
This section of our documentation details the core concepts used to create quantum programs in Q#, and to interact with those programs from classical applications.
```